### PR TITLE
Updating my personal page.

### PR DESCRIPTION
**Pull request by nickwhitman1993**
@rtrp/admins @ryanstwrt @aaronjamesreynolds 

Let me know if you see any huge errors.

### CODE REVIEW

**Comment by ryanstwrt** [comment]
@nickwhitman1993 I'm not able to view the figures when I go to your page. Are you able to see them on your end?

**Comment by nickwhitman1993** [comment]
@ryanstwrt yeah, that sounds about right. If you go into the commit and look at the files it should show up though.

File: users/whitmann/index.md [hunk 1]
```diff
@@ -25,50 +25,75 @@ My current research involves solving the radiative transport equation for phonon
...
 * ***Los Alamos National Laboratory (LANL)*** **W-Division Student Poster Presentation** *Los Alamos, New Mexico* August, 2016
 
 ***
 
 ## Work Experience
-* ***TerraPower, LLC*** , Mentors: Jacob Hader & Oded Doron, *June - September, 2019*
-  * **Nuclear Design Group - Shielding Methods and Analysis**
+* ***Los Alamos National Laboratory (LANL)***
+  * **XCP-3: Monte Carlo Methods, Codes, and Applications**, Mentors: Travis Trahan (XCP-3) and Jesson Hutchinson (NEN-2), *June - September 2020*
+    * Verification and validation of the Subcritical Reflected alpha-Plutonium (SCRaP) experiments performed at NCERC using the Monte Carlo Application ToolKit (MCATK) software. The SCRaP experiments involved sixteen configurations of an alpha-phase, weapons grade Plutonium sphere surrounded by varying layers of copper and polyethylene reflectors and two Helium-3 multiplicity counters. Analysis included comparisons of MCATK to MCNP and measured values of multiplication factor, multiplication, count rates, and other counting parameters such as Feynman variance-to-mean.
```
Comment (aaronjamesreynolds) [inline users/whitmann/index.md]: weapons**-**grade **p**lutonium

Comment (aaronjamesreynolds) [reply users/whitmann/index.md]: **h**elium-3

Comment (aaronjamesreynolds) [reply users/whitmann/index.md]: I may not have the background, but are multiplication factor and multiplication different things? Just checking :)

Comment (aaronjamesreynolds) [reply users/whitmann/index.md]: Everything else looks good! I wish there was a better way to make sure images would show up on the reviewers' end of things, but alas.

Comment (nickwhitman1993) [reply users/whitmann/index.md]: Thanks for the edits - and multiplication can be interpreted as the average length of fission chains and related to multiplication factor by M = 1/k-1

Comment (aaronjamesreynolds) [reply users/whitmann/index.md]: Ahh, I see! The more you know.

Comment (ryanstwrt) [reply users/whitmann/index.md]: It's looking good to me too. Once those edits are updated any of us can probably merge the request?

Comment (aaronjamesreynolds) [reply users/whitmann/index.md]: Yup yup!

Comment (nickwhitman1993) [reply users/whitmann/index.md]: Believe I fixed this. I will merge.

**Pull request merged**